randovania/data/json_data/prime3.json

```diff
@@ -417,7 +417,603 @@
+            {
+                "index": 21,
+                "long_name": "Gateway Platform near Crash Site",
+                "short_name": "Event21"
+            },
+            {
+                "index": 22,
+                "long_name": "Gateway Orange Lock",
+                "short_name": "Event22"
+            },
+            {
+                "index": 23,
+                "long_name": "Gateway Blue Lock",
+                "short_name": "Event23"
+            },
+            {
+                "index": 24,
+                "long_name": "Gateway Tunnel Orb",
+                "short_name": "Event24"
+            },
+            {
+                "index": 25,
+                "long_name": "Grand Court Path Phazon Vines",
+                "short_name": "Event25"
+            },
+            {
+                "index": 26,
+                "long_name": "Grand Court Statue",
+                "short_name": "Event26"
+            },
+            {
+                "index": 27,
+                "long_name": "Grand Court Tunnel Panels",
+                "short_name": "Event27"
+            },
+            {
+                "index": 28,
+                "long_name": "Grand Court Golem",
+                "short_name": "Event28"
+            },
+            {
+                "index": 29,
+                "long_name": "Hillside Vista Statue",
+                "short_name": "Event29"
+            },
+            {
+                "index": 30,
+                "long_name": "GFC Theseus Satellite",
+                "short_name": "Event30"
+            },
+            {
+                "index": 31,
+                "long_name": "Hidden Court Golem",
+                "short_name": "Event31"
+            },
+            {
+                "index": 32,
+                "long_name": "Hidden Court Burrow Gate",
+                "short_name": "Event32"
+            },
+            {
+                "index": 33,
+                "long_name": "Fuel Gel Pool Golem Head",
+                "short_name": "Event33"
+            },
+            {
+                "index": 34,
+                "long_name": "Hangar Bay Power",
+                "short_name": "Event34"
+            },
+            {
+                "index": 35,
+                "long_name": "Hangar Bay Ceiling",
+                "short_name": "Event35"
+            },
+            {
+                "index": 36,
+                "long_name": "Hangar Bay Ship Upgrade",
+                "short_name": "Event36"
+            },
+            {
+                "index": 37,
+                "long_name": "Hall of the Golems - Right Golem",
+                "short_name": "Event37"
+            },
+            {
+                "index": 38,
+                "long_name": "Hall of the Golems - Left Golem",
+                "short_name": "Event38"
+            },
+            {
+                "index": 39,
+                "long_name": "Hall of the Golems - Top Golem",
+                "short_name": "Event39"
```

Comment:
Not super important, but it would be more clear if the golems were named after weapon needed (Plasma, Ice Missile, Grapple Voltage)